ts/image-occlusion/mask-editor.ts

```diff
@@ -120,8 +120,12 @@ function initCanvas(onChange: () => void): fabric.Canvas {
     return canvas;
 }
 
-const getImageData = (imageData): string => {
+const getImageData = (imageData, path): string => {
     const b64encoded = protoBase64.enc(imageData);
+    const extension = path.split(".").pop();
+    if (extension === "svg") {
+        return "data:image/svg+xml;base64," + b64encoded;
+    }
     return "data:image/png;base64," + b64encoded;
```

Comment:
Ok, so we've been lying about the content type :-) While the browser tolerates this for things like .jpg images, we should probably be trying to return the correct content type for jpg/webp/gif as well.

Reply:
Yeah, I have added more mime type.